functions/go/format/metadata.yaml

```diff
@@ -0,0 +1,11 @@
+image: gcr.io/kpt-fn/fmt
```

Comment:
Should this be `gcr.io/kpt-fn/format`?
The directory name should match the image name.

Reply:
Done.